NineChronicles.DataProvider/DataRendering/AuraSummonData.cs

```diff
@@ -18,13 +21,22 @@ public static class AuraSummonData
             long blockIndex
         )
         {
+            var prevAura = previousStates.GetAvatarStateV2(avatarAddress).inventory.Equipments
+                .Where(e => e.ItemSubType == ItemSubType.Aura).Select(e => e.ItemId);
+            var gained = outputStates.GetAvatarStateV2(avatarAddress).inventory.Equipments
+                .Where(e => e.ItemSubType == ItemSubType.Aura && !prevAura.Contains(e.ItemId)).Select(e => e.Id);
+            Console.WriteLine(gained);
```

Comment:
@U-lis Should we remove this part because it's not used to derive `gainedAura`?

Reply:
Or maybe we can change `gainedAura` to:

```
 var gainedAura = string.Join(",", gained);
```

Reply:
Oh, I've forgot to remove codes for debug. I'll clean it up.